src/odemis/driver/andorcam3.py

```diff
@@ -354,7 +353,6 @@ def __init__(self, name, role, device=None, bitflow_install_dirs=None,
             tran_rng = ((-uh_shape[0], -uh_shape[1]),
                         (uh_shape[0], uh_shape[1]))
             self.translation = model.ResolutionVA((0, 0), tran_rng, unit="px",
-                                                  cls=(int, long),
```

Comment:
Note: we can completely drop the "cls" argument, because the default value does the same job (better).